_dev/src/views/debug.vue

```diff
@@ -154,6 +154,49 @@
       </b-card-body>
     </b-card>
 
+    <b-card
+      no-body
+      class="ps_gs-onboardingcard px-0"
+    >
+      <b-card-header
+        header-tag="h3"
+        header-class="px-3 py-3 font-weight-600 ps_gs-fz-16 mb-0"
+      >
+        ProductFeed infos
+      </b-card-header>
+      <b-card-body
+        body-class="p-3"
+      >
+        <ul class="mb-0">
+          <li>
+            <strong>ProductFeed status</strong>:
+            migration : {{ productFeed.migration ?'✅':'❌' }}
+            fullSync : {{ productFeed.fullSync ?'✅':'❌' }}
+          </li>
+          <li>
+            <b-button
+              class="mt-3 mr-3"
+              variant="primary"
+              @click="sendMigration(true)"
+              :disabled="!GET_PRODUCT_FEED_SETTINGS"
+            >
+              Enabled migration
```

Comment:
Are we talking about an action to trigger?

```suggestion
              Enable migration
```